test/test.c

```diff
@@ -1983,6 +1988,12 @@ static int ParseOptions( int argc, char ** argv )
                     {
                         free( input );
                         input = malloc( strlen( "/dev/" ) + strlen( devName ) + 1 );
+                        if( input == NULL )
+                        {
+                            fprintf( stderr, "ERROR: malloc() failed while attempting to set DVD device path.\n" );
```

Comment:
Nit: Remove "DVD" from the error message since this applies to any optical device. One day we'll change the function name. ;)